specification/dartLangSpec.tex

```diff
@@ -8969,24 +8969,28 @@ \subsection{Symbols}
 a valid declaration name or a valid library name in a Dart program.
 
 \begin{grammar}
-<symbolLiteral> ::= `#' (<operator> | (<identifier> (`.' <identifier>)*))
+<symbolLiteral> ::=
+  `#' (<operator> | <identifier> (`.' <identifier>)* | \VOID{})
 \end{grammar}
 
 \LMHash{}%
 A symbol literal \code{\#\id} where \id{} is an identifier
-that does not begin with an underscore (`\code{\_}'),
-evaluates to an instance of \code{Symbol} representing the identifier \id.
-All occurrences of \code{\#\id} evaluate to the same instance
```

Comment:
I'd use commas: "\id, respectively \VOID,". 
Or, looking at other usages,, out ", respectively." last: 
> representing the identifier \id{} or the reserved word \VOID{}, respectively

Reply:
This was rewritten.